### ISSUE TITLE
Filter large files

### ISSUE THREAD

**Issue by dmitryuk** [wagoodman/dive#440]
After I run dive I can see thousands of changes. Can I filter it by file/directory size? Or it will be great to have the feature?

**Comment by mark2185** [comment]
You can currently only filter by path, by pressing `ctrl+F` in the `Current Layer Contents` view.

Maybe collapsing dirs with `Space` will help.

**Comment by mark2185** [comment]
I agree, unfortunately the project is a bit inactive, so unless someone contributes to a fork somewhere, I'm afraid it is what it is.
